util/gh-pages/script.js

```diff
@@ -579,6 +579,40 @@ function setTheme(theme, store) {
     }
 }
 
+function getVirtualKey(ev) {
+    if ("key" in ev && typeof ev.key !== "undefined") {
+        return ev.key;
+    }
+
+    const c = ev.charCode || ev.keyCode;
+    if (c === 27) {
+        return "Escape";
+    }
+    return String.fromCharCode(c);
+}
+
+function handleShortcut(ev) {
+    if (document.activeElement.tagName === "INPUT") {
+        if (getVirtualKey(ev) === "Escape") {
+            document.activeElement.blur();
+        }
+    } else {
+        switch (getVirtualKey(ev)) {
+            case "s":
+            case "S":
+            case "/":
+                ev.preventDefault(); // To prevent the kep to be put into the input.
```

Comment:
typo - key

Reply:
We probably should add an early return here also if there's a modifier key on, it doesn't seem super likely but someone could want to `ctrl + s` to save the page

Reply:
Very good point! Adding that.